addons/mail/static/src/components/follower_list_menu/follower_list_menu.js

```diff
@@ -16,6 +16,7 @@ class FollowerListMenu extends Component {
      */
     constructor(...args) {
         super(...args);
+        this.sprintf = _.str.sprintf;
```

Comment:
same remarks

Reply:
Done, added a computed field also in thread model to depend on "followers" field.